utilities/cli/holohub.py

```diff
@@ -196,6 +197,9 @@ def _create_parser(self) -> argparse.ArgumentParser:
         run.add_argument(
             "--language", choices=["cpp", "python"], help="Specify language implementation"
         )
+        run.add_argument(
+            "--run_args", help="Additional arguments to pass to the application executable"
```

Comment:
Can we avoid the --run_args and just use any extra arguments?

`./holohub run myapp --test`